packages/lib/preprocess/index.cjs

```diff
@@ -32,7 +33,7 @@ module.exports = function evidencePreprocess(componentDevelopmentMode = false) {
 						'*': 'markdown'
 					}
 				],
-				[rehypeSlug],
+				rehypeSlug,
```

Comment:
Is this change related?

Reply:
@ItsMeBrianD 
square brackets are required to supply options at index 1, which we are not supplying here
ref: https://mdsvex.pngwn.io/docs#remarkplugins--rehypeplugins